python/capypdf.py

```diff
@@ -1686,12 +1687,13 @@ def set_destination(self, destination):
 
     def set_uri(self, uri):
         uritxt = uri.encode('ASCII')
-        check_error(libfile.capy_annotation_set_uri(self, uritxt))
+        check_error(libfile.capy_annotation_set_uri(self, uritxt, len(uritxt)))
```

Comment:
Do we know if python strings are null term or are you being cautious with adding the len?

Reply:
Python bytestrings in ctypes are guaranteed to be null terminated. I added the length because it is known and it increases performance because you don't have to first compute the length (the practical difference is probably immeasurably small).